docs/basic_training/operators.pt.md

```diff
@@ -0,0 +1,271 @@
+---
+description: Material de treinamento básico do Nextflow
+---
+
+# Operadores
+
+Operadores são métodos para conectar canais, transformar valores emitidos por canais ou executar regras próprias em canais.
+
+Existem sete grupos de operadores descritos em detalhe na Documentação do Nextflow, estes são:
+
+1. [Operadores de filtragem](https://www.nextflow.io/docs/latest/operator.html#filtering-operators)
+2. [Operadores de transformação](https://www.nextflow.io/docs/latest/operator.html#transforming-operators)
+3. [Operadores de divisão](https://www.nextflow.io/docs/latest/operator.html#splitting-operators)
+4. [Operadores de combinação](https://www.nextflow.io/docs/latest/operator.html#combining-operators)
+5. [Operadores de bifurcação](https://www.nextflow.io/docs/latest/operator.html#forking-operators)
+6. [Operadores matemáticos](https://www.nextflow.io/docs/latest/operator.html#maths-operators)
+7. [Outros operadores](https://www.nextflow.io/docs/latest/operator.html#other-operators)
```

Comment:
Is this accurate? The sections in the website seem to be different (e.g. Reduction, Parsing Text Data)

Reply:
Good point. It's outdated, but we can't do anything from the translation side. I will fix it on the original document, but before we must set up a system that will automatically flag outdated translations. I will keep this one to test the system once it's set up.

Reply:
Agreed!